wagtail/admin/panels/inline_panel.py

```diff
@@ -26,7 +27,7 @@ def __init__(
         super().__init__(*args, **kwargs)
         self.relation_name = relation_name
         self.panels = panels
-        self.heading = heading or label
+        self.heading = heading or label or capfirst(relation_name.replace("_", " "))
```

Comment:
I saw this pattern in a few other places, might be worth extracting as its own function in a separate PR.